docs/predictors/inferenceservice-cr.md

```diff
@@ -0,0 +1,126 @@
+  - `BlockedByFailedLoad` - The predictor's currently active model configuration is older than its current spec reflects, because there was a problem loading the corresponding model. See the `lastFailureInfo` field for more details of the failure.
+  - `InvalidSpec` - The predictor's currently active model configuration was not transitioned to match its current spec because the current spec is invalid. There may be more details of the error in the `lastFailureInfo` field.
+
+- `modelCopies` - Model copy information of the predictor's model.
+
+  - `failedCopies` - The number of copies of the active or target model that failed to load recently (there will be at most one of each per pod).
+  - `totalCopies` - The total number copies of this predictor's models that are currently loaded.
+
+- `lastFailureInfo` - Details about the most recent error associated with this predictor. Not all of the contained fields will necessarily have a value.
+
+  - `reason` - A high level code indicating the nature of the failure, may be one of:
+    - `ModelLoadFailed` - The model failed to load within a serving runtime container. Loading is automatically retried in other pods of the same runtime if they exist, the `failedCopies` field indicates how many different pods the model _recently_ failed to load in.
+    - `RuntimeUnhealthy` - Corresponding `ServingRuntime` pods failed to start or are unhealthy.
+    - `NoSupportingRuntime` - There are no `ServingRuntime`s which support the specified model type.
+    - `RuntimeNotRecognized` - There is no `ServingRuntime` defined with the specified runtime name.
+    - `InvalidPredictorSpec` - The current `InferenceService` predictor spec is invalid or unsupported.
+  - `location` - Indication of the pod in which a loading failure most recently occurred, if applicable. Its value will be the last 12 digits of the pod's full name.
+  - `message` - A message containing more detail about the error/failure.
+  - `modelId` - The internal id of the model in question. This includes a hash of the InferenceService's predictor spec.
+  - `time` - The time at which the failure occurred, if applicable.
+
+Upon creation, InferenceService the model status will always transition to `Loaded` state (unless the loading fails), but later if unused it is possible that they end up in a `Standby` state which means they are still available to serve requests but the first request could incur a loading delay. Whether this happens is a function of the available capacity and usage pattern of other models. It's possible that models will transition from `Standby` back to `Loaded` "by themselves" if more capacity becomes available.
```

Comment:
Maybe just me, not quite sure
1. what "InferenceService the model status" means. I suppose it means the model status of the InferenceService?
2. what "they" refer to in "...they end up in a `Standby` state which means they are still available to serve requests..." I suppose they means InferenceServices?

Reply:
Yea, it's not quite clear here. I will reword.